docs/source/using/addons.adoc

```diff
@@ -273,5 +273,19 @@ After you define the add-on, you can install it by running:
 $ minishift addons install <ADDON_DIR_PATH>
 ----
 
+TIP: You can also write metadata in multiple line.
+[[example-multiline-metadata]]
+.Example: Add-on definition which contain multiline description
+
```

Comment:
the indentation is not right, when rendered. The actual example is not part of the TIP anymore, but rendered in the surrounding context. One needs a '+' to connect the content within the TIP in this case. 

There is really no alternative to actually rendering the Asciidoc to see whether the syntax is correct and as expected. 

Simple preview of some editors make it sometimes look like correct, but it is not.

I recommend to install the Ruby tool chain. 'rake serve' takes no time at all to spin up. The Docker setup is mainly there to provide a Happy Path. The better experience is to have the Ruby toolchain installed to run the Rake tasks directly. @minishift/minishift-dev please try. The editing experience when using 'rake serve' in conjunction with a live reload enabled browser is unbeatable. You edit, you save, you look at your bowser and you see the rendered output fully integrated into the site.